src/lib/model/InboundTransfersModel.js

```diff
@@ -207,13 +249,26 @@ class InboundTransfersModel {
                 if (now > expiration) {
                     const error = Errors.MojaloopApiErrorObjectFromCode(Errors.MojaloopApiErrorCodes.QUOTE_EXPIRED);
                     this._logger.error(`Error in prepareTransfer: quote expired for transfer ${prepareRequest.transferId}, system time=${now} > quote time=${expiration}`);
+                    // now store the fulfilment and the quote data against the quoteId in our cache
+                    await this._cache.set(`tf_${prepareRequest.transferId}`, {
+                        request: prepareRequest,
+                        mojaloopResponse: error,
+                    });
                     return this._mojaloopRequests.putTransfersError(prepareRequest.transferId, error, sourceFspId);
                 }
             }
 
             // project the incoming transfer prepare into an internal transfer request
             const internalForm = shared.mojaloopPrepareToInternalTransfer(prepareRequest, quote);
 
+
+            await this._cache.set(`tf_${prepareRequest.transferId}`, {
+                request: prepareRequest,
+                mojaloopResponse: {
+                    transferState: 'RESERVED',
```

Comment:
@kirgene the DFSP backend must be the single source of truth for transfer state; the cache in the SDK should not really be thought of as that point of truth.